research-software-directory/RSD-as-a-service
# Docker improvements

# Docker improvements

Fixes #

Changes proposed in this pull request:

* Use two-stage build for authentication module
* Upgrade various backend dependencies

How to test:

Run the usual build flow:
* `docker-compose down --volumes`
* `docker-compose build`
* `docker-compose up`
* Everything should work as before

PR Checklist:
